### PR TITLE
[WebAPI] Add FingerPrint tests to demo-express repo

### PR DESCRIPTION
**Pull request by qiuzhong**
FingerPrint usecase can be a good demo to be integrated in demo-express
repo.

BUG=https://crosswalk-project.org/jira/browse/CTS-615

### CODE REVIEW

**Comment by qiuzhong** [comment]
@Honry , please take a review and tell if any license or COPYING files missing.

**Comment by qiuzhong** [comment]
@Honry , fingerprint example updated.

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ DemoExpress is a central place to collect web feature samples that demonstrate u
   * Networking & Storage: `FileReader`, `IndexedDB`, `Session History`, `Web Database`, `Web Messaging`, `Web Speech`, `Web Storage`
   * Performance & Optimization: `Animation Timing`,  `High Resolution Time`, `Navigation Timing`, `Performance Timeline`, `Resource Timing`, `Selectors`, `Typed Array`, `User Timing`, `Workers`
   * Device & Hardware: `Accelerometer`, `Browser State`, `Camera via UserMedia`, `Full Screen`, `Forms`, `HTML Template`, `Input`, `Notifications`, `Screen Orientation`, `Touch`, `WebAudio`, `WebGL`
-  * Experimental: `Device Capabilities`, `Presentation`, `SIMD`
+  * Experimental: `Device Capabilities`, `Presentation`, `SIMD`, `Fingerprint`
```
Comment (Honry) [inline README.md]: ‘s/Fingerprint/FingerPrint’

File: samples/FingerPrint/js/fingerprint.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+function authenticate() {
```
Comment (Honry) [inline samples/FingerPrint/js/fingerprint.js]: Please add license.

**Comment by Honry** [comment]
LGTM

**Pull request merged**